src/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
 cmake_minimum_required(VERSION 2.8)
 
+CMAKE_POLICY(SET CMP0079 NEW)
+
+include_directories(/usr/local/include)
+include_directories(/usr/local/Cellar/libusb-compat/0.1.5_1/include)
+include_directories(/usr/local/opt/readline/include)
+
+link_directories(/usr/local/lib)
+link_directories(/usr/local/Cellar/libusb-compat/0.1.5_1/lib)
+link_directories(/usr/local/opt/readline/lib)
+
```

Comment:
Sorry for just getting to this!

Is there any slightly more portable way of identifying brew include/ld directories? I know this is a super bit-rotted repository, but I'd rather still avoid hard-coding paths :)

Reply:
Yes, that makes sense. This was a fast fix for the issue that took me a lot of time to solve. I might not be able to comment on brew installations because I only used it once when doing the setup for this.

Reply:
ok! I'll just leave this open so people can easily find and apply it if they want. Should I/anyone ever pick up maintenance again, it can be resolved properly then.

Just glad my random college tinkerings were useful to someone!

Reply:
Yeah. That works fine. Yes, there was a project I was doing for someone some time back and he wanted me to use this.